homeassistant/components/logbook/__init__.py

```diff
@@ -444,12 +444,9 @@ def yield_events(query):
         return list(humanify(hass, yield_events(query), entity_attr_cache, prev_states))
 
 
-def _keep_event(hass, event, entities_filter, entity_attr_cache):
+def _keep_event(hass, event, entities_filter):
     if event.event_type == EVENT_STATE_CHANGED:
         entity_id = event.entity_id
-        if entity_id is None:
```

Comment:
`entity_id` will never be none here because we already excluded it in sql if its a EVENT_STATE_CHANGED